### PR TITLE
Initial setup to add package to artifactory

### PR DESCRIPTION
**Pull request by MaxOSullivan**
Initial setup and changes for moving the slate-edit-table to the home office and to artifactory

### CODE REVIEW

File: .drone-1.0.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+steps:
+  - name: install
+    image: node:16
+    environment:
+      NPM_AUTH_USERNAME:
+        from_secret: npm_auth_username
+      NPM_AUTH_TOKEN:
+        from_secret: npm_auth_token
+    commands:
+      - npm ci
```
Comment (lennym) [inline .drone-1.0.yml:14]: Since there is a test script, it should be run.

```suggestion
      - npm ci
      - npm test
```

Comment (MaxOSullivan) [reply .drone-1.0.yml:14]: Have added that, will now work on fixing the 66 broken tests

Comment (lennym) [reply .drone-1.0.yml:14]: 😬

Comment (lennym) [reply .drone-1.0.yml:14]: Feel free to remove it if it's a pain. I had assumed the tests would be passing.

**Pull request merged**